lib/wallaby/phantom.ex

```diff
@@ -1,13 +1,45 @@
 defmodule Wallaby.Phantom do
+  @moduledoc """
+  Wallaby driver for PhantomJS.
+
+  ## Usage
+
+  Start a Wallaby Session using this driver with the following command:
+
+  ```
+  {:ok, session} = Wallaby.start_session(driver: Wallaby.Phantom)
+  ```
+
+  ## Notes
+
+  This driver requires PhantomJS be installed in your path. You can install PhantomJS through NPM or your package manager of choice:
```

Comment:
I'd say that it requires it in the path or you configure a custom path. As for the example I think it's something like `node_modules/bin/phantomjs` or something :)